File: problems/easy/programmer_day/__tests__/test_programmer_day.js
const { dayOfProgrammer } = require("../day_of_programmer");

const year = 1984;
const result = dayOfProgrammer(year);

if (result !== "12.09.1984") {
    throw new Error(`❌ Test failed. Expected 12.09.1984, got ${result}`);
}

console.log("✅ JavaScript test passed!");

File: problems/easy/programmer_day/__tests__/test_programmer_day.cpp
#include <iostream>
#include <string>
#include "../programmer_day.cpp"

void test() {
    int year = 1984;
    string result = dayOfProgrammer(year);

    if (result != "12.09.1984") {
        std::cerr << "❌ Test failed. Expected 12.09.1984, got " << result << "\n";
        exit(1);
    }

    std::cout << "✅ C++ test passed!\n";
}

int main() {
    test();
    return 0;
}
